QATCH/processors/Elaborate.py

```
@@ -70,58 +102,59 @@ def __init__(self, queue_log, parser_process, queue_in, queue_out, export, overt
+                    sequence,
+                    timestamp,
+                    peak_mag,
+                    peak_freq,
+                    left,
+                    right,
+                    data_temp,
+                    baseline,
+                    overtone,
+                )
 
         except:
```

Ruff (0.15.13)

[error] Do not use bare `except`

(E722)